### PR TITLE
MES-6905 VRN Waiting Room & Comms Screen

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Description
Introduces the VRN modal, including a button to access it on waiting room and comms screen
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image
image
image
image

### CODE REVIEW

File: capacitor.config.json [hunk 1]
```diff
@@ -10,4 +10,4 @@
     }
   },
   "cordova": {}
-}
+}
```
Comment (Cullima) [inline capacitor.config.json]: can you revert

File: src/components/common/vrn-capture-modal/vrn-capture-modal.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  .save-button {
+    color: var(--gds-black);
+  }
+
+  input[type="text"] {
+    font-size: 18px;
+  }
+}
```
Comment (Cullima) [inline src/components/common/vrn-capture-modal/vrn-capture-modal.scss]: can you add missing line

File: src/components/common/vrn-capture-modal/vrn-capture-modal.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  vehicleRegistrationFormControlName: string = 'vehicleRegistration';
+
+  formInvalid: boolean = false;
+
+  readonly registrationNumberValidator: FieldValidators = getRegistrationNumberValidator();
+
+  constructor(
+    private navParams: NavParams,
+  ) {
+    this.onCancel = this.navParams.get('onCancel');
```
Comment (Cullima) [inline src/components/common/vrn-capture-modal/vrn-capture-modal.ts]: not sure this is the recommended way of passing data between page and modal.  Think we should be using `dismiss()` and `onDidDismiss()` methods (see [docs](https://ionicframework.com/docs/api/modal#dismissing-a-modal)) without need for `navParams`

**Review by Cullima — CHANGES_REQUESTED** [review]
Looks good but couple of small changes required I think.  Also, can you looka t re-populating input with any previously saved value from store if re-entering page / resuming a test?

Also needs some screen grabs

**Review by Cullima — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**